$NetBSD: patch-ad,v 1.1 2002/11/19 20:54:21 agc Exp $

Remove -malign-double from configuration options - causes problems in
at least the stat structure - from Christian Biere

--- configure.in	2002/11/19 20:37:03	1.1
+++ configure.in	2002/11/19 20:37:57
@@ -123,12 +123,6 @@
         AC_MSG_RESULT(Turning on gcc 3 optimisations)
         CFLAGS="$CFLAGS -minline-all-stringops -fno-math-errno"
       fi
-      case $target_cpu in
-        i?86) AC_MSG_RESULT(Turning on pentium optimisations)
-              CFLAGS="$CFLAGS -march=pentium -malign-double -malign-loops=5"
-              CFLAGS="$CFLAGS -malign-jumps=5 -malign-functions=5"
-              ;;
-      esac
     else
       AC_MSG_WARN(You did not opt for gcc optimisations!)
       optimum=no
